packages/okta-react/test/e2e/harness/e2e/page-objects/protected.po.js

```diff
@@ -14,11 +14,11 @@ import { browser, element, by, ExpectedConditions } from 'protractor';
 
 export class ProtectedPage {
   navigateTo() {
-    return browser.get('/protected');
+    return browser.get('/protected?foo=bar#baz');
```

Comment:
If possible, lets try and keep [the same page objects](https://github.com/okta/okta-oidc-js/blob/master/packages/okta-angular/test/e2e/harness/e2e/page-objects/protected.po.ts#L22) throughout our front-end integrations (as we want to use shared specs in the future).

Can this be updated to `/protected/foo?state=bar`? This will allow us to test:
1. Nested paths
2. Retained query params.

Reply:
I'd propose `/protected/foo?state=bar#baz` then because I also want to assert we preserve fragments (we had issues in the past where they were removed, I think it was an authJS issue that got resolved)